docs/src/docs/asciidoc/user/tiering.adoc

```diff
@@ -88,31 +179,98 @@ With the above into account, the following possibilities are valid configuration
 
 * heap + offheap
 * heap + offheap + disk
-* heap + disk
 * heap + offheap + clustered
+* heap + disk
 * heap + clustered
 
-== Persistent tiers
+Here is an example using heap, offheap and clustered.
 
-The _disk_ and _clustered_ tiers introduce the concept of persistence.
-It means that the data they contain can survive a restart of the Java application hosting the cache instance.
+[source,java,indent=0]
+----
+include::{sourcedir33}/clustered/client/src/test/java/org/ehcache/clustered/client/docs/Tiering.java[tag=threeTiersCacheManager]
+----
 
-Persistent cluster tier::
-The cluster tier is persistent since the data lives in a different JVM.
-However the open source clustering implementation does not preserve data across restarts of the server JVM hosting the Terracotta cluster.
+== Resource pools
 
-Persistent disk tier::
-By default the disk tier is non persistent.
-It will wipe the data files upon clean shutdown or during restart.
-If you want the disk tier to be persistent, it needs to be configured as such, see examples below.
-Note that the disk tier persistence is not crash proof.
-It requires a clean shutdown of the cache manager through the invocation of `CacheManager.close()`.
+Tiers are configured using resource pools. Most of the time using a `ResourcePoolsBuilder`. Let's revisit an example from
+the <<getting-started.adoc#three-tiers, Getting Started>>.
 
 [source,java,indent=0]
 ----
-include::{sourcedir33}/impl/src/test/java/org/ehcache/docs/Tiering.java[tag=diskPersistent]
+include::{sourcedir33}/impl/src/test/java/org/ehcache/docs/GettingStarted.java[tag=threeTiersCacheManager]
```

Comment:
Since this fragment has callouts, it looks weird in the rendered doc. Why not duplicate it once in code in a version with less or no callouts?
After all, the reason we keep these in code is to make sure they get hit by a refactoring. But I do not mind at all have a duplicate code sections between two docs source files.